tmp/build/generate-manifests.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+# to generate Maistra OLM metadata: MAISTRA_VERSION=1.0.0 DEPLOYMENT_FILE=deploy/maistra-operator.yaml MANIFESTS_DIR=manifests-maistra tmp/build/generate-manifests.sh
+# to generate ServiceMesh OLM metadata: MAISTRA_VERSION=1.0.0 DEPLOYMENT_FILE=deploy/servicemesh-operator.yaml MANIFESTS_DIR=manifests-servicemesh tmp/build/generate-manifests.sh
+
+set -e
+
+: ${DEPLOYMENT_FILE:?"Need to set location of the source deployment file, e.g. deploy/maistra-operator.yaml"}
+: ${MANIFESTS_DIR:?"Need to set location of the manifests directory, e.g. manifests/"}
+: ${MAISTRA_VERSION:?"Need to set maistra version, e.g. 1.0.0"}
+
+BUNDLE_DIR=${MANIFESTS_DIR}/${MAISTRA_VERSION}
+MY_LOCATION="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+mkdir -p "$BUNDLE_DIR"
+
+function checkDependencies() {
```

Comment:
is there a specific python version required?

Reply:
Not that I know of, I'm using default versions of everything when it comes to python install.